source/guide_littlelinkcustom.rst

```diff
@@ -0,0 +1,110 @@
+.. highlight:: console
+
+.. author:: Jan Klomp <https://klomp.de>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: lang-php
+.. tag:: web
+.. tag:: self-hosting
+.. tag:: analytics
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/littlelinkcustom.svg
+      :align: center
+
+##########
+LittleLink Custom
+##########
+
+.. tag_list::
+
+LittleLinkCustom_ is a highly customizable link sharing and landing page platform with an easy to use interface and acts as a free opensource alternative to Linktree, Shorby, bio.fm etc.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+    * :manual:`PHP <lang-php>`
+    * :manual:`domains <web-domains>`
+
+License
+=======
+
+*LittleLink Custom* is licensed under the GNU General Public License v3.0. All relevant legal information can be found here:
+
+  * https://github.com/JulianPrieber/littlelink-custom/blob/main/LICENSE
+
+Prerequisites
+=============
+
+We're using :manual:`PHP <lang-php>` in the stable version 8.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '8.1'
+ [isabell@stardust ~]$
+
+If you want to use *LittleLink Custom* with your own domain you need to setup your :manual:`domains <web-domains>` first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your :manual:`document root <web-documentroot>`, then download the latest release of *LittleLink Custom* and extract it:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/JulianPrieber/littlelink-custom/releases/latest/download/littlelink-custom.zip
+ ...
+ [isabell@stardust html]$ unzip littlelink-custom.zip
+ ...
+ [isabell@stardust ~]$
+
+Move the extracted files and directories to the document root, then delete the empty source directory, the downloaded zip file and the ``nocontent.html`` file:
+
+::
+
+ [isabell@stardust ~]$ mv littlelink-custom/* littlelink-custom/.[!.]* .
+ [isabell@stardust ~]$
+ [isabell@stardust ~]$ rm littlelink-custom.zip
+ [isabell@stardust ~]$
+ [isabell@stardust ~]$ rm -r littlelink-custom
+ [isabell@stardust ~]$
+ [isabell@stardust ~]$ rm nocontent.html
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Open your webbrowser and go to ``https://isabell.uber.space/``, then log in to your installation with the default login ``admin@admin.com`` and password ``12345678``.
```

Comment:
it looks like the setup way has changed in between, you have to choose a DB etc. Please advise before to create DB `isabell_littlelink` (lookup other guides for the corresponding CREATE code) and choose that one in the web installer.

If there is a way to execute the installation not in web but on the terminal that would be a better way for the guide

Reply:
@noave, thank you for the hints!

I've uploaded the missing image file for the logo, changed the database setup process to MySQL and adjusted the installation process to your change request.

Hope the guide is ok now (I've tested it with success on a fresh installation). Please let me know if there is anything else that I can do.

Thanks!